### PR TITLE
Fix/outbound foreign keys

### PR DESCRIPTION
**Pull request by somebody32**
## What

LHM has no support for foreign keys, me and @sauloperez wanted to add that support, but looks like it was done already, but not merged: https://github.com/soundcloud/lhm/pull/77, so I've just backported that.

Also added additional check to drop all foreign keys for renamed tables when cleaning up (useful when migrating from the old lhm)

### CODE REVIEW

File: lib/lhm.rb [hunk 1]
```diff
@@ -108,4 +122,15 @@ def connection
     Connection.new(adapter)
   end
 
+  def clean_up_consraints(table_name)
```
Comment (sauloperez) [inline lib/lhm.rb]: That's the thing we added

Comment (somebody32) [inline lib/lhm.rb]: There was a huge merge, so we need to check everything

**Comment by saimonmoore** [comment]
@somebody32 works great. :postal_horn: && :+1:

**Comment by somebody32** [comment]
@saimonmoore addressed everything

**Pull request merged**